### DISCUSSION TITLE
FNF but bad

### DISCUSSION THREAD

**Discussion by Intancote** [Saltssaumure/fnf-osu-mania-skin#24] [Ideas]
I like these skins a lot and was wondering if there is gonna be even a but bad version just for funnies.
Even a Shaggy/Circle version of the but bad skin would be cool.
I know it's a lot to ask but it's just an idea, it doesn't have to happen.

**Comment by Saltssaumure** [comment]
You're (and others) are more than welcome to make this a reality. The skin elements are simply `.png` images, so they are able to be edited by anyone using any image editing software.
I personally use Medibang Paint Pro (free!). There are other options such as Krita, GIMP, Paint.net too.
